exasol-s3/src/main/java/com/exasol/spark/s3/ExasolWriteBuilderProvider.java

```diff
@@ -99,6 +99,7 @@ private LogicalWriteInfo getUpdatedLogicalWriteInfo(final LogicalWriteInfo defau
         final Map<String, String> map = new HashMap<>(defaultInfo.options().asCaseSensitiveMap());
         map.put("header", "true");
         map.put("delimiter", ",");
+        map.put("fs.s3a.committer.name", "magic");
```

Comment:
Do you plan to update the version of the SEC in the pom file?
Depending on the version (unchanged or new version) please update changes file describing the changes with reference to the issue #163.
See folder [doc/changes](https://github.com/exasol/spark-connector/tree/refactoring/%23163-custom-output-committer/doc/changes), files `changelog.md` and `changes_*.md`.
When changing the version number in the pom file PK will update file `changelog.md` and create file `changes_*.md`.

Reply:
Yes, agree with @ckunki, please create another minor version upgrade for refactoring and update the change files.

All else looks good, thanks looking into it :+1:

Reply:
This should also fix the failing PK verification CI action.

Reply:
Did version increment and `pk fix`, which generated the changelog and dependency file.

But it didn't help with PK CI check: https://github.com/exasol/spark-connector/actions/runs/5879323540/job/15943123992?pr=193